tests/integration/components/live-participants-test.js

```diff
@@ -116,6 +116,7 @@ module('Integration | Component | live-participants', function (hooks) {
     />`);
     assert.dom('[data-test-sidebar-body-role-guest]').hasText('Mavens (2)');
     assert.dom('[data-test-sidebar-user="2"]').hasText('Maven1');
+    assert.dom('.user').exists('data-test-icon');
```

Comment:
usually we do like this,
assert.dom('element').exists();

Here what you're passing in exists are not getting that element.

Reply:
commented https://github.com/Real-Dev-Squad/website-www/pull/692#discussion_r1356520075